apps/web/playwright/fixtures/users.ts

```diff
@@ -145,12 +145,12 @@ export async function login(
 }
 
 export async function getPaymentCredential(page: Page) {
-  await page.goto("/apps/installed");
+  await page.goto("/apps/stripe");
```

Comment:
You can't install anything from "Installed Apps" anymore, making it so it goes directly to Stripe app page